helm/hpcc/values.yaml

```diff
@@ -748,6 +758,50 @@ roxie:
   #directAccessPlanes: []  #add direct access planes that roxie will read from without copying the data to its default data plane
   #ldapUser: roxie_file_access    #add system username for accessing files
   #egress: engineEgress
+  hpa:
```

Comment:
@jakesmith this PR isn't done yet, and I don't intend to include this set of sample HPA metrics, but it illustrates the concept. It would all us to provide sensible default HPAs for various workload resources. Take a look a provide any feedback you might have. thanks